pkg/gateway/model_build_targetgroup.go

```diff
@@ -372,6 +391,23 @@ func (t *latticeServiceModelBuildTask) buildTargetGroupSpec(ctx context.Context,
 
 	tgName := latticestore.TargetGroupName(string(httpBackendRef.Name()), namespace)
 
+	tgp, err := getAttachedTargetGroupPolicy(ctx, client, string(httpBackendRef.Name()), namespace)
```

Comment:
following code block looks identical to line 198-213

Reply:
it is right now, but will be quite different with GRPC serviceexport follow-up change